ProcessMaker/Http/Controllers/Api/TaskController.php

```diff
@@ -191,9 +202,9 @@ public function index(Request $request, $getTotal = false, User $user = null)
             try {
                 $query->pmql($pmql, null, $user);
             } catch (QueryException $e) {
-                return response(['message' => __('Your PMQL search could not be completed.')], 400);
+                abort('Your PMQL search could not be completed.', 400);
```

Comment:
Does this change affect the display of messages?

Reply:
This is a SonnarQueue observations because this funtion 5 return and only 3 are permitted